src/librustc_back/target/mipsel_unknown_linux_musl.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright 2014 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use target::Target;
+
+pub fn target() -> Target {
+    Target {
+        llvm_target: "mipsel-unknown-linux-musl".to_string(),
+        target_endian: "little".to_string(),
+        target_pointer_width: "32".to_string(),
+        arch: "mips".to_string(),
```

Comment:
> Should this be mipsel?

Not sure! The existing `mipsel-unknown-linux-gnu` target is also using `mips` as the arch.

AIUI, this arch field is used for `#[cfg(target_arch = ...)]` and not for codegen (that's what llvm-target is used for). If we want to change both targets (mipsel-gnu and mipsel-musl) to use arch = mipsel, then we'll have to update the cfgs in crates like libc from `#[cfg(target_arch = "mips")]` to `#[cfg(any(target_arch = "mips", any(target_arch = "mipsel"))]`.